lib/compilers/clspv.ts

```diff
@@ -90,7 +90,7 @@ export class CLSPVCompiler extends BaseCompiler {
     override optionsForFilter(filters, outputFilename) {
         const sourceDir = path.dirname(outputFilename);
         const spvBinFilename = path.join(sourceDir, this.outputFilebase + '.spv');
```

Comment:
Shouldn't this be factored out in `getOutputFilename` ?

Reply:
done